AsyncDisplayKit/ASMapNode.mm

```diff
@@ -307,6 +318,15 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
   return size;
 }
 
+- (void)calculatedLayoutDidChange
+{
+  [super calculatedLayoutDidChange];
+  
+  if (_snapshotNeeded) {
+    [self takeSnapshot];
```

Comment:
Can you explain why that is the case? 

As far as I can tell, this would only increase the frequency with which the snapshot is taken. The base class implementation is empty, so the largest impact that this method could have is additional calls to snapshot. 

As far as I can tell, there is no case where this would be able to prevent a `takeSnapshot` call that would have occurred previously.